distribution/changelog.txt

```diff
@@ -2,6 +2,7 @@
 ------------------------------------------------------------------------
 - Feature: [#3868] Initial support for using TTF in OpenGL mode (still contains bugs).
 - Feature: [#7682] Follow ride/guest/staff in main window viewport.
+- Feature: [#11262] Speed property for vehicle animations in .parkobj
```

Comment:
https://github.com/OpenRCT2/OpenRCT2/issues/11262 has nothing to do with this.

Reply:
This was a typo